Domain/Wine.cs

```diff
@@ -0,0 +1,16 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace WineCellar.Domain;
+
+public class Wine
+{
+    public int Id { get; set; }
+
+    [Required]
```

Comment:
we use fluent validation. https://github.com/CarterCommunity/Carter

i examples viser de hvordan kan man bruge fluent validation til at validere modeller